### ISSUE TITLE
Contact's suffix and prefix and malformed phone number

### ISSUE THREAD

**Issue by myleneb** [CaliOpen/Caliopen#672]
When you create a new contact, suffix and prefix are not considered.
Example: when you create a contact and only fill `suffix` field with `o`, it returns:
```
{
family_name:""
given_name:""
name_prefix:""
name_suffix:""
.....
title:"o"
```

When you create a contact with a malformed phone number, phone number isn't saved.

**Comment by gdchamal** [comment]
Some validation occu on phone number :

```
apiv1_1            | 2017-11-24 13:28:12,520 WARNI [caliopen_main.contact.objects.phone][waitress] Unable to normalize phone number 01234567 : (0) Missing or invalid default region.
apiv1_1            | 2017-11-24 13:28:12,533 INFO  [caliopen_main.common.objects.base][waitress] document is not valid: {"phones": {"0": {"normalized_number": ["The string supplied did not seem to be a phone number."]}}}
apiv1_1            | 2017-11-24 13:28:12,534 INFO  [caliopen_main.common.objects.base][waitress] ObjectIndexable apply_patch() returned error: document is not valid, can't insert it into db: <{"phones": {"0": {"normalized_number": ["The string supplied did not seem to be a phone number."]}}}>
```

**Comment by gdchamal** [comment]
Fixed with 0.6.0 release
